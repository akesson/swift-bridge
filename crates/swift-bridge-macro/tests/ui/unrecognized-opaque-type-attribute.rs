//! # To Run
//! cargo test -p swift-bridge-macro -- ui trybuild=unrecognized-opaque-type-attribute.rs

// <!-- ANCHOR: mdbook-ui-test-example -->
#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        #[swift_bridge(InvalidAttribute)]
        type SomeType;
    }
}

pub struct SomeType;
// <!-- ANCHOR_END: mdbook-ui-test-example -->

fn main() {}
