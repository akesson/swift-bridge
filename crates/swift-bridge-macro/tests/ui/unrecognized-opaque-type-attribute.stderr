error: Unrecognized attribute "InvalidAttribute".
 --> tests/ui/unrecognized-opaque-type-attribute.rs:8:24
  |
8 |         #[swift_bridge(InvalidAttribute)]
  |                        ^^^^^^^^^^^^^^^^
